### PR TITLE
add check test result for interface net1

### PR DESCRIPTION
**Pull request by yulng**
add check test result for interface net1

@s1061123

### CODE REVIEW

File: e2e/test-default-route1.sh [hunk 1]
```diff
@@ -7,7 +7,13 @@ kubectl create -f yamls/default-route1.yml
 kubectl wait --for=condition=ready -l app=default-route1 --timeout=300s pod
 
 echo "check default-route-worker1 interface: net1"
-kubectl exec default-route-worker1 -- ip a show dev net1
+net=$(kubectl exec default-route-worker1 -- ip a show dev net1)
```
Comment (s1061123) [inline e2e/test-default-route1.sh:10]: the variable `net` seems not to used anywhere. could you please let me know why `net` is required?

File: e2e/test-default-route1.sh [hunk 2]
```diff
@@ -24,7 +30,13 @@ if [ $ipaddr != "10.1.1.254" ]; then
 fi
 
 echo "check default-route-worker2 interface: net1"
-kubectl exec default-route-worker2 -- ip a show dev net1
+net2=$(kubectl exec default-route-worker2 -- ip a show dev net1)
```
Comment (s1061123) [inline e2e/test-default-route1.sh:33]: the variable `net2` seems not to used anywhere. could you please let me know why `net` is required?

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: e2e/test-default-route1.sh (hunk 1 above)
Comment (yulng) [reply e2e/test-default-route1.sh:10]: ok, echo $net

File: e2e/test-default-route1.sh (hunk 2 above)
Comment (yulng) [reply e2e/test-default-route1.sh:33]: ok, echo $net2

Comment (yulng) [reply e2e/test-default-route1.sh:33]: @s1061123  😅

File: e2e/test-default-route1.sh (hunk 1 above)
Comment (s1061123) [reply e2e/test-default-route1.sh:10]: We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?

File: e2e/test-default-route1.sh (hunk 2 above)
Comment (s1061123) [reply e2e/test-default-route1.sh:33]: We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: e2e/test-default-route1.sh (hunk 1 above)
Comment (yulng) [reply e2e/test-default-route1.sh:10]: > We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?
ok,updated it , kubectl output will be printed
image

Comment (s1061123) [reply e2e/test-default-route1.sh:10]: This code still seems to be redundant. how about that?

```
kubectl exec default-route-worker1 -- ip a show dev net1
if [ $? -ne 0 ];then
       exit 1
fi
```

File: e2e/test-default-route1.sh (hunk 2 above)
Comment (s1061123) [reply e2e/test-default-route1.sh:33]: This code still seems to be redundant. how about that?

```
kubectl exec default-route-worker2 -- ip a show dev net1
if [ $? -ne 0 ];then
       exit 1
fi
```

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: e2e/test-default-route1.sh (hunk 2 above)
Comment (yulng) [reply e2e/test-default-route1.sh]: ok,updated

**Review by s1061123 — APPROVED** [review]
LGTM. merged.

**Pull request merged**